.github/workflows/ci-workflow.yaml

```diff
@@ -0,0 +1,53 @@
+name: ci-workflow
+
+on:
+  [push, workflow_dispatch]
+
+jobs:
+
+  build:
+    name: Build and Package
+    runs-on: ubuntu-latest 
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: Run a Start script line
+        run: echo Start api-spec Build
+
+      - name: Set up adoptopenjdk8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: 'gradle'
+      
+      - name: run openapi Validate
+        uses: mbowman100/swagger-validator-action@master
+        with:
+          files: |
+            management-api/management.yml
+      
+      - name: set Branch env var
```

Comment:
Nit -- some capitalization inconsistencies in the step names. (E.g. "run" and "step" are each capitalized in one name and not in another. )

Reply:
I have made these more consistent now.  

See latest.